patroni/postgresql/postmaster.py

```diff
@@ -105,6 +105,29 @@ def from_pid(cls, pid):
         except psutil.NoSuchProcess:
             return None
 
+    def signal_kill(self):
+        """to suspend and kill postmaster and all children
+
+        :returns True if postmaster and children are killed, False if error
+        """
+        try:
+            self.suspend()
+            children = self.children()
+            self.kill()
+        except psutil.NoSuchProcess:
+            pass
+        except psutil.AccessDenied as e:
+            logger.warning("Could not kill PostgreSQL (error: {0})".format(e))
+            return False
+        _, alive = psutil.wait_procs(children, timeout=0)
+        for child in alive:
+            try:
+                child.kill()
+            except psutil.NoSuchProcess:
+                continue
+
```

Comment:
I wonder, should we wait for processes before returning from the method?
```python
psutil.wait_procs(children + [self])
```

Reply:
It probably helps in case of Zombie state.

Reply:
Please review the following changes. Thanks.

Reply:
Fixed